stl/inc/xlocale

```diff
@@ -651,6 +651,49 @@ protected:
     }
 };
 
+template <class _Elem>
+constexpr bool _One_byte_char_like() {
+    return false;
+}
+
+template <>
+constexpr bool _One_byte_char_like<char>() {
+    return true;
+}
+
+template <>
+constexpr bool _One_byte_char_like<signed char>() {
+    return true;
+}
+
+template <>
+constexpr bool _One_byte_char_like<unsigned char>() {
+    return true;
+}
+
+#ifdef __cpp_lib_byte
+template <>
+constexpr bool _One_byte_char_like<byte>() {
+    return true;
+}
+#endif // __cpp_lib_byte
+
+#ifdef __cpp_char8_t
+template <>
+constexpr bool _One_byte_char_like<char8_t>() {

```

Comment:
I'm not sure if specializations is the right thing. Maybe `if constexpr` with `is_same` or `_Is_any_of_v` ?